### DISCUSSION TITLE
Bond Hotspot via wifi on Slate

### DISCUSSION THREAD

**Discussion by loggerhead-turtle** [SmoothWAN/SmoothWAN#90] [General & Help]
I don't know why I can't get the slate to bond connections that are wired and wireless.  For instance, I have my home connection connected via WAN and I want to connect the Slate to my mobile hotspot via wifi (not via the wired connection).  How does one do this?  I can't seem to figure it out for the life of me.

Essentially, I'm trying to get around the issue with the iPhone not being able to share a network connection, due to the inability to "trust" the slate. I want to be able to share cell connections from my iPhone to my slate.

Thanks!

**Comment by TalalMash** [comment]
For the iPhone is `Allow others to join` enabled?
![image](https://user-images.githubusercontent.com/96490382/212331774-7c3730ac-30d8-433e-a141-64f4741aacf7.png)
Do you see `iphone` or a new `ethx` in _Network -> Interfaces -> Devices (tab)_?
Next version will automatically connect the latest iOS version since they changed the name to `iphone` and that is not automated with the current version.

For SmoothWAN v0.99.8 and older you need to tap disable before scanning, next version will allow it to scan regardless:
![image](https://user-images.githubusercontent.com/96490382/212332891-e7fd361a-dd06-4329-bcce-b27a652cd927.png)
![image](https://user-images.githubusercontent.com/96490382/212333277-e79ae6ec-29f5-45e5-a4b3-ed4413addbac.png)
![image](https://user-images.githubusercontent.com/96490382/212333410-4c300082-4efc-42a9-aa70-2ff3dc56be27.png)
